OrrRock/OrrRock/ViewControllers/VideoCollectionViewController/VideoCollectionViewController.swift

```diff
@@ -239,21 +246,31 @@ class VideoCollectionViewController: UIViewController {
     }
     
     @objc func didSelectAllButtonClicked(_ sender: UIBarButtonItem){
-        if videoInformationArray.count != 0{
-            for i in 0...videoInformationArray.count-1{
-                let indexPath = IndexPath(item: i, section: 0)
-                dictionarySelectedIndexPath.updateValue(true, forKey: indexPath)
+        if isFirstSelectAllButtonTouch{
+            if videoInformationArray.count != 0{
+                for i in 0...videoInformationArray.count-1{
+                    let indexPath = IndexPath(item: i, section: 0)
+                    dictionarySelectedIndexPath.updateValue(true, forKey: indexPath)
+                }
+                for row in 0..<self.videoCollectionView.numberOfItems(inSection: 0) {
+                    self.videoCollectionView.selectItem(at: IndexPath(row: row, section: 0), animated: false, scrollPosition: [])
+                }
             }
-            for row in 0..<self.videoCollectionView.numberOfItems(inSection: 0) {
-                self.videoCollectionView.selectItem(at: IndexPath(row: row, section: 0), animated: false, scrollPosition: [])
+        }
+        else{
+            for (key,value) in dictionarySelectedIndexPath{
```

Comment:
```suggestion
            for (key,value) in dictionarySelectedIndexPath {
```

p1;

`{` 는 한 칸을 띄워주세요! 위 아래의 코드들도 반영해주세요.

Reply:
반영했습니다!